Clayton-TV/claytontv
# Basic search functionality

## Description

Allows basic searching of database by video name and description

Relates to #56

## Type of change


## How has this been tested?

1. Search for videos using the search bar in the navigation bar.
2. Search results page is loaded (without fully reloading the browser window)
3. Any videos in the database with matching names show first, then those with matching descriptions.

This was developed on top of my branch matt/video-view-page, but split into a new branch as the functionality is not directly related.
Currently on this branch the results page is more basic, because that branch updated the video list vue components.
Once PR #64 is merged the results page title and description will appear properly.

## Checklist:



## What should reviewers look for?

Please provide any specific areas of feedback you're looking for from reviewers.
